client/main.lua

```diff
@@ -182,7 +153,9 @@ local function initPrison(time)
 	CurrentJob = "Electrician"
 	CreateJobBlip()
 	applyClothes()
-	createCellsBlip()
+	createCellsBlip(Config.Locations.yard.coords, 238, locale("info.cells_blip"), CellsBlip, 0.8)
+	createCellsBlip(Config.Locations.freedom.coords, 466, locale("info.freedom_blip"), TimeBlip, 0.8)
+	createCellsBlip(Config.Locations.shop.coords, 52, locale("info.canteen_blip"), ShopBlip, 0.8)
```

Comment:
The function createCellsBlip now returns the blip in question, but you don't assign it to anything yet. Aside from that, add a colour argument as well to differentiate the colour of the blips if desired.